workers/src/main/scala/workers/CatalogCache.scala

```diff
@@ -115,7 +122,9 @@ trait CatalogCache extends CatalogIDB with AsyncToIO {
     // Make a time based query for pm over a year
     val query = TimeRangeQueryByADQL(
       tracking,
-      Bounded(start.toInstant(Constants.UTCOffset), end.toInstant(Constants.UTCOffset), 0),
+      Interval
+        .closed(start.toInstant(Constants.UTCOffset), end.toInstant(Constants.UTCOffset))
+        .asInstanceOf[Bounded[Instant]],
```

Comment:
Maybe a pattern match would be better here?

Reply:
Good idea, done